packages/buidler-core/src/internal/cli/cli.ts

```diff
@@ -181,6 +185,15 @@ async function main() {
 
     console.log("");
 
+    const reporter = Reporter.getInstance();
+    try {
+      if (config === undefined || config.analytics.enabled) {
```

Comment:
This seems weird, but the point is that if an error was thrown before the config could be loaded, we err on the side of reporting it.

Reply:
I see. This makes lots of sense. I would just add a comment explaining it, so we don't accidentally delete it in the future.